lib/lambda_ethereum_consensus/logger/console_logger.ex

```diff
@@ -0,0 +1,46 @@
+defmodule ConsoleLogger do
+  @moduledoc """
+  Custom logger formatter for console output.
+  """
+
+  def format(level, message, timestamp, metadata) do
+    pattern = Logger.Formatter.compile(" $time $message ")
+
+    formatted_metadata =
+      Enum.map_join(
+        metadata,
+        " ",
+        fn {key, value} ->
+          IO.ANSI.bright() <>
+            Atom.to_string(key) <> IO.ANSI.reset() <> "=" <> format_metadata(key, value)
+        end
+      )
+
+    [format_level(level)] ++
+      [Logger.Formatter.format(pattern, level, message, timestamp, [])] ++
+      [formatted_metadata] ++ ["\n"]
+  rescue
+    err ->
+      inspect(err)
```

Comment:
Why this?

Reply:
If there is an error in the logger code, then it gets swallowed otherwise. With this, at least we log the error of the logger code.